torch/cuda/profiler.py

```diff
@@ -50,6 +50,11 @@ def start():
 
 
 def stop():
+    r"""Stops cuda profiler data collection
+    
```

Comment:
Looks like lint failure is real, are there trailing spaces here?

Reply:
yes. Thank you for spotting the issue. I have fixed the format now.